vega-lite-schema.json

```diff
@@ -2036,43 +2036,13 @@
                     "type": "string"
                 },
                 "shape": {
-                    "anyOf": [
-                        {
-                            "enum": [
-                                "circle",
-                                "cross",
-                                "diamond",
-                                "square",
-                                "triangle-down",
-                                "triangle-up"
-                            ],
-                            "type": "string"
-                        },
-                        {
-                            "type": "string"
-                        }
-                    ],
-                    "description": "The default symbol shape to use. One of: `\"circle\"` (default), `\"square\"`, `\"cross\"`, `\"diamond\"`, `\"triangle-up\"`, or `\"triangle-down\"`, or a custom SVG path."
+                    "description": "The default symbol shape to use. One of: `\"circle\"` (default), `\"square\"`, `\"cross\"`, `\"diamond\"`, `\"triangle-up\"`, or `\"triangle-down\"`, or a custom SVG path.",
+                    "type": "string"
```

Comment:
@kanitw This is a actually a reasonable optimization but I wanted to call it out to you.